src/Management/src/Endpoint/ManagementWebHostBuilderExtensions.cs

```diff
@@ -372,12 +371,8 @@ private static IConfiguration GetConfigurationFallback()
 
         try
         {
-            var environment = Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT");
-            config = new ConfigurationBuilder()
```

Comment:
The left side seems better to me dont you think? Can we fix it to allow multi line and not long streams of code?

Reply:
I would guess this was an over-correction for the blank line on 379. We have plenty of examples of multi-line statements like this (just without extra lines in between)

Reply:
This is a known limitation. The Resharper formatter cannot distinguish between Fluent API syntax and regular chained method calls, such as: `GetItems().Any()`. As explained [here](https://github.com/bkoelman/CSharpGuidelinesAnalyzer/issues/129#issuecomment-1111028535), this is a tough problem to tackle for any code analyzer.

However, it can be overridden on a case-by-case basis as below:

```c#
// @formatter:wrap_chained_method_calls chop_always
// @formatter:keep_existing_linebreaks true

            config = new ConfigurationBuilder()
                .AddJsonFile("appsettings.json")
                .AddJsonFile($"appsettings.{environment}.json", optional: true)
                .Build();

// @formatter:keep_existing_linebreaks restore
// @formatter:wrap_chained_method_calls restore
```